apps/judicial-system/backend/src/app/formatters/requestPdf.ts

```diff
@@ -65,11 +72,11 @@ function constructRestrictionRequestPdf(
     .lineGap(40)
     .text(`Dómstóll: ${existingCase.court?.name}`, { align: 'center' })
     .font('Helvetica-Bold')
-    .fontSize(14)
+    .fontSize(mediumFontSize)
     .lineGap(8)
     .text(formatMessage(m.baseInfo.heading))
     .font('Helvetica')
-    .fontSize(12)
+    .fontSize(baseFontSize)
```

Comment:
This will change from font size 12 to font size 11.
Is that what we want here?

Reply:
Same ☝️